### PR TITLE
fix(@desktop/wallet): Update the link to point to terms and condition of paraswap page.

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
### What does the PR do

Update the link to point to terms and condition of paraswap page.

### Affected areas

SwapSignModal

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://github.com/user-attachments/assets/70ef9196-81bd-4d1f-b035-f2d30d216308

### CODE REVIEW

**Review by dlipicar — APPROVED** [review]

File: ui/app/AppLayouts/Wallet/popups/swap/SwapSignModal.qml [hunk 1]
```diff
@@ -42,8 +42,8 @@ SignTransactionModalBase {
     required property string cryptoFees
     required property double slippage
 
-    property string serviceProviderName: "Paraswap"
-    property string serviceProviderURL: "https://www.paraswap.io/" // TODO https://github.com/status-im/status-desktop/issues/15329
+    property string serviceProviderName
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/popups/swap/SwapSignModal.qml:45]: We might as well make them `required` now, wdyt?

**Review by caybro — APPROVED** [review]
LGTM of course

File: ui/app/AppLayouts/Wallet/popups/swap/SwapSignModal.qml (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/Wallet/popups/swap/SwapSignModal.qml:45]: make sense!

Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/Wallet/popups/swap/SwapSignModal.qml]: done!

**Comment by virginiabalducci** [comment]
Looks good! QA passed @Khushboo-dev-cpp

**Pull request merged**